From 05f31ce8c605a564b049242230c3e22d8ca30a30 Mon Sep 17 00:00:00 2001
From: Dan McDonald <danmcd@joyent.com>
Date: Fri, 8 Sep 2017 08:37:44 -0400
Subject: [PATCH] OS-6334 OS-816 forgot about ipsecah Reviewed by: Jason King
 <jason.king@joyent.com> Approved by: Robert Mustacchi <rm@joyent.com>

---
 usr/src/uts/common/inet/ip/ipsecah.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/usr/src/uts/common/inet/ip/ipsecah.c b/usr/src/uts/common/inet/ip/ipsecah.c
index b533d75398..7d5eac71b3 100644
--- a/usr/src/uts/common/inet/ip/ipsecah.c
+++ b/usr/src/uts/common/inet/ip/ipsecah.c
@@ -173,8 +173,8 @@ ah_kstat_init(ipsecah_stack_t *ahstack, netstackid_t stackid)
 	ipsec_stack_t	*ipss = ahstack->ipsecah_netstack->netstack_ipsec;
 
 	ahstack->ah_ksp = kstat_create_netstack("ipsecah", 0, "ah_stat", "net",
-	    KSTAT_TYPE_NAMED, sizeof (ah_kstats_t) / sizeof (kstat_named_t),
-	    KSTAT_FLAG_PERSISTENT, stackid);
+	    KSTAT_TYPE_NAMED, sizeof (ah_kstats_t) / sizeof (kstat_named_t), 0,
+	    stackid);
 
 	if (ahstack->ah_ksp == NULL || ahstack->ah_ksp->ks_data == NULL)
 		return (B_FALSE);
-- 
2.21.0

